app/src/main/kotlin/com/wire/android/ui/common/BottomNavigation.kt

```diff
@@ -54,17 +53,16 @@ fun WireBottomNavigationBar(
     ) {
         Row(horizontalArrangement = Arrangement.SpaceBetween) {
             items.forEachIndexed { index, item ->
+                val padding = MaterialTheme.wireDimensions.bottomNavigationPadding
                 val modifier = Modifier
                     .weight(1f)
                     .fillMaxHeight()
                     .background(MaterialTheme.colorScheme.surface)
                     .padding(
-                        top = Dimensions.bottomNavigationPadding,
-                        bottom = Dimensions.bottomNavigationPadding,
-                        start = Dimensions.bottomNavigationPadding
-                                + if (index == 0) Dimensions.bottomNavigationPadding else spaceBetweenItems / 2,
-                        end = Dimensions.bottomNavigationPadding
-                                + if (index == items.lastIndex) Dimensions.bottomNavigationPadding else spaceBetweenItems / 2
+                        top = padding,
+                        bottom = padding,
+                        start = padding + if (index == 0) padding else spaceBetweenItems / 2,
+                        end = padding + if (index == items.lastIndex) padding else spaceBetweenItems / 2
```

Comment:
little bit hard for me to read what is happening here, like why are you divding the spaceBetweenItems / 2 , maybe you could extract it to some private method to make it more readable ?

Reply:
I extracted these values to `WireDimensions` - we'll be able to change it depending on screen dimensions